config/environment.js

```diff
@@ -1,5 +1,10 @@
 var url = require('url');
 
+function isTrue(value) {
+  value = String(value).toLowerCase();
+  return value === 'true';
+}
+
```

Comment:
@cosmosgenius why not make this a helper/util?

Reply:
May be will move when we used it ones more